### ISSUE TITLE
Page modification request: mv ls example

### ISSUE THREAD

**Issue by jxu** [tldr-pages/tldr#13414]
### Command description

    ls | parallel|xargs mv -t path/to/target_directory

Last example has bad practice parsing ls https://unix.stackexchange.com/questions/128985/why-not-parse-ls-and-what-to-do-instead 
(plus, the two different uses of pipe character here are confusing)

#13353 @fazlearefin you should know this is bad practice. Plus I don't see what this adds over example 3

  - Move multiple files into an existing directory, keeping the filenames unchanged:
    mv path/to/source1 path/to/source2 ... path/to/existing_directory

### Platform

Common

**Comment by fazlearefin** [comment]
How about you send PRs with the perfect examples rather than calling out people?

> two different uses of pipe character here are confusing

Two different pipes? I suggest you spend more time learning how to read tldr docs.

**Comment by gutjuri** [comment]
I agree. I propose to modify the example as to read:

```bash
find . -maxdepth 1 -print0 | xargs -0 mv -t {{path/to/target_directory}}
```

Do you know whether or not this would work for filenames containing spaces? I can't test right now.

**Comment by fazlearefin** [comment]
`print0` will handle spaces fine.

**Comment by jxu** [comment]
If you're ok with  ignoring hidden files, you can probably do 

    mv * /target

> I agree. I propose to modify the example as to read:
> 
> ```shell
> find . -maxdepth 1 -print0 | xargs -0 mv -t {{path/to/target_directory}}
> ```
> 
> Do you know whether or not this would work for filenames containing spaces? I can't test right now.

If you use find, you can avoid xargs with something like

    find . -maxdepth1 -exec mv {} /target \; 

but this will try to move `.` as well

**Comment by fazlearefin** [comment]
@jxu 

What we are trying to do here is trying to show how to use the `-t` flag in `mv` command. Your examples repeat what is already available in `find` command.

So if you want to help, come up with examples that show the use of `-t` in `mv`.

**Comment by jxu** [comment]
I've never used it (or anything other than -v) and I suggest leaving it off

**Comment by gutjuri** [comment]
> I've never used it (or anything other than -v) and I suggest leaving it off

Actually, I've also never used it. I didn't even know that `-t` existed before today. So yes, maybe removing it from the page is better than having an unrealistic example like the one that I proposed :P

**Comment by fazlearefin** [comment]
There are situations I had to use `-t`. 

I have updated the example to:

```
{{find /var/log -type f -name '*.log' -print0}} | {{xargs -0}} mv -t {{path/to/target_directory}}
```

Yes, it is possible to use `find` alone to achieve the task but using `-exec` along with `find` is complex for many users (the trailing `\;` is very strange for beginners). TLDR examples should target all users and give them flexibility.

**Comment by jxu** [comment]
> TLDR examples should target all users and give them flexibility.

I agree somewhat, but tldr is mainly focused on common usecases. If an example is too contrived, then it probably doesn't belong as an example.
